cmd/buildah/images.go

```diff
@@ -246,12 +246,11 @@ func outputImages(images []storage.Image, format string, store storage.Store, fi
 				ID:        image.ID,
 				Name:      name,
 				Digest:    digest,
-				CreatedAt: createdTime,
+				CreatedAt: createdTime.Format("Jan 2, 2006 15:04"),
```

Comment:
The godoc for the `time` package explains that the values of the numeric fields are used to determine which field in a reference date structure is meant to be placed at that location in the output string.